components/collector/ci/quality.sh

```diff
@@ -15,10 +15,9 @@ run mypy src
 run pylint --rcfile=../../.pylintrc src tests
 run python -m flake8 --select=DUO src  # Dlint
 unset PYTHONDEVMODE  # Suppress ResourceWarnings given by pip-audit in dev mode
-run pip-audit --strict --progress-spinner=off -r requirements/requirements-base.txt -r requirements/requirements.txt -r requirements/requirements-dev.txt
+run pip-audit --strict --progress-spinner=off -r requirements/requirements.txt -r requirements/requirements-dev.txt
 export PYTHONDEVMODE=1
-run safety check --bare -r requirements/requirements-base.txt -r requirements/requirements.txt -r requirements/requirements-dev.txt
+run safety check --bare -r pyproject.toml -r requirements/requirements.txt -r requirements/requirements-dev.txt
```

Comment:
Is this not superfluous because the two requirements files are seeded by pyproject.toml?